tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -310,8 +315,13 @@ constexpr bool optional_test() {
     }
     {
         constexpr std::optional o1(SmallVal);
+        constexpr derived_optional<decltype(SmallVal)> derived1{std::optional(SmallVal)};
+        constexpr derived_optional<decltype(SmallVal)> derived2{std::optional(LargeVal)};

```

Comment:
No change requested: I observe that these temporaries could be constructed with braces, following our preferred modern convention. Although this test uses braces to construct temporaries elsewhere, I can't quite bring myself to reset testing for such a small issue in test code (which certainly doesn't affect correctness or test coverage).